modules/base/src/components/Base.vue

```diff
@@ -193,48 +250,8 @@ export default {
       this.uiModules[0].loaded = true;
       this.activeModule = this.uiModules[0].url;
```

Comment:
is `this.uiModules[0]` a safe assumption?

Reply:
Yes, that is the module of the subsystem itself.